### PR TITLE
Rewrote the document raw_data.md

### PR DESCRIPTION
**Pull request by Astarsys**
Added tables for columns instead of text
Added table for get_inputs response
Added more information about every endpoint, parameter, example and errors 
Added required column into request parameters table. It is responsible for parameter required/optional
Changed headers a bit to be more precise
Added information about Parquet format file - parameter, description and example.

### CODE REVIEW

File: docs/data-warehouse-api/resources/tracker/raw_data.md [hunk 1]
```diff
@@ -71,90 +84,127 @@ curl -X 'POST' \
...
 
-!!! note "Instead of using `from`/`to` parameters it is possible to set `interval` parameter - ISO 8601 formatted interval, for example 2023-08-24T08:04:36.306Z/PT24H."
+!!! note "Instead of using the `from`/`to` parameters, you can set the `interval` parameter — an ISO 8601 formatted interval, for example, `2023-08-24T08:04:36.306Z/PT24H`."
+
+The response could be provided in a CSV or [Parquet](https://parquet.apache.org/docs/overview/) format file, with columns 
+predefined in the `columns` parameter of the API request. 
 
-The response is provided in a CSV format file, with columns that are predefined in the `columns` parameter of the API request.
 Here are the specifications for the table output:
```
Comment (john9x) [inline docs/data-warehouse-api/resources/tracker/raw_data.md:138]: @Astarsys these specifications related only to CSV format

Comment (Astarsys) [reply docs/data-warehouse-api/resources/tracker/raw_data.md:138]: Yes, thank you. I will add this to my next commit

File: docs/data-warehouse-api/resources/tracker/raw_data.md [hunk 2]
```diff
@@ -71,90 +84,127 @@ curl -X 'POST' \
...
 * Values are separated by a comma character `,`.
 * Rows are separated using the Unix line feed (LF).
 * `NULL` is represented as `\N`.
 * Requested column can be a simple or complex.
 
-**Simple** columns:
-
-* `msg_time` - time of message sent by the device. Always returned in CSV output and does not need to be requested separately. Indicated according to user account time zone.
-* `server_time` - time of message processing by the server.
-* `gps_fix_type` - enum. One of `UNKNOWN`, `NO_FIX`, `HAS_FIX`, `LAST_KNOWN_POSITION`.
```
Comment (john9x) [inline docs/data-warehouse-api/resources/tracker/raw_data.md:125]: @Astarsys also, in the parquet format enums represented as integers

gps_fix_type - UNKNOWN=0, NO_FIX=1, HAS_FIX=2, LAST_KNOWN_POSITION=3
mn_type - UNKNOWN=0, GSM=1, CDMA=2, WCDMA=3, LTE=4, NR=5)

Comment (Astarsys) [reply docs/data-warehouse-api/resources/tracker/raw_data.md:125]: sure, I added it under the table.

Comment (john9x) [reply docs/data-warehouse-api/resources/tracker/raw_data.md:125]: oops, sorry

**Pull request merged**